src/lib/txVerify.js

```diff
@@ -17,6 +17,59 @@ const TransactionInconsistentReason = {
   RECIPIENT_CRYPTO_ADDRESS_MISMATCH: 'recipient_crypto_address_mismatch'
 }
 
+export async function checkTxInProcess(api, context, withoutNonce = true, externalNonce = null) {
+  const fetchInfo = CryptosInfo[context.state.crypto].txFetchInfo
+  const TX_LOCK_TIME = fetchInfo.newPendingAttempts * fetchInfo.newPendingInterval
```

Comment:
According to the specification the `txFetchInfo` and its properties are optional. Take this into consideration and perform additional check.

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/c7856ba3-5166-45a8-bef5-bf6586837a1d)